### PR TITLE
We will cal you

### PR DESCRIPTION
**Pull request by Valentan390**

### CODE REVIEW

File: src/css/base/common.css [hunk 1]
```diff
@@ -15,7 +15,7 @@ ol {
 }
 
 body {
-  font-family: 'DM', sans-regular;
+  font-family: 'DM Sans', sans-regular;
```
Comment (guemplen) [inline src/css/base/common.css:18]: This file is shared, it cannot be modified or committed! The font has already been added to the main branch.

File: src/css/layout/we-will-call-you.css [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+  background-color: var(--green);
+}
+.div-call-you {
+  max-width: 335px;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: #eaf7e5;
+  border-radius: 40px;
+  padding-top: 31px;
+  background-image: url(../../img/we-will-call-you/BG.png);
```
Comment (guemplen) [inline src/css/layout/we-will-call-you.css:81]: There is an extra dot in the path to the image file.

File: src/css/layout/we-will-call-you.css [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: -23px 5px;
+}
+@media (min-device-pixel-ratio: 2),
+  (min-resolution: 192dpi),
+  (min-resolution: 2dppx) {
+  .div-call-you {
+    background-image: url(../../img/we-will-call-you/BG@2x.png);
```
Comment (guemplen) [inline src/css/layout/we-will-call-you.css:90]: There is an extra dot in the path to the image file.

File: src/css/layout/we-will-call-you.css [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+    padding: 15px 35px;
+  }
+  .div-call-you {
+    border-radius: 80px;
+    padding-top: 17px;
+    padding-left: 38px;
+    padding-right: 38px;
+    max-width: 704px;
+    background-position: -15px 9px;
+    background-image: url(../../img/we-will-call-you/BG-TAB.png);
```
Comment (guemplen) [inline src/css/layout/we-will-call-you.css:126]: There is an extra dot in the path to the image file.

File: src/css/layout/we-will-call-you.css [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+  }
+  @media (min-device-pixel-ratio: 2),
+    (min-resolution: 192dpi),
+    (min-resolution: 2dppx) {
+    .div-call-you {
+      background-image: url(../../img/we-will-call-you/BG-TAB@2x.png);
```
Comment (guemplen) [inline src/css/layout/we-will-call-you.css:132]: There is an extra dot in the path to the image file.

File: src/css/layout/we-will-call-you.css [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+  }
+  .div-call-you {
+    padding-top: 37px;
+    padding-left: 0;
+    padding-right: 0;
+    margin-left: 0;
+    margin-right: 0;
+    width: calc((100% - 77px) / 2);
+    background-position: 4px 0;
+    background-image: url(../../img/we-will-call-you/BG-desk.png);
```
Comment (guemplen) [inline src/css/layout/we-will-call-you.css:180]: There is an extra dot in the path to the image file.

File: src/css/layout/we-will-call-you.css [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+  }
+  @media (min-device-pixel-ratio: 2),
+    (min-resolution: 192dpi),
+    (min-resolution: 2dppx) {
+    .div-call-you {
+      background-image: url(../../img/we-will-call-you/BG-desk@2x.png);
```
Comment (guemplen) [inline src/css/layout/we-will-call-you.css:186]: There is an extra dot in the path to the image file.

**Review by guemplen — CHANGES_REQUESTED** [review]
Please, before making a pull request, commit only the files of your modules and images!!!

**Comment by Valentan390** [comment]
Удалил лишние точки

**Pull request closed without merging**